Google-Summer-of-Code-2025.md

```diff
@@ -43,6 +43,18 @@ Furthermore, note that GSoC isn't just about code -- it's also about communicati
 You might also enjoy the "weekly journals" from some of our previous contributors: **[Rd4dev](https://medium.com/@rd4dev)** and **[@theMr17](https://medium.com/@Mr_17)**.
 
 
+## Selected Projects
```

Comment:
@Nik-09 Could you please include links to their journals as well?

Reply:
Done